meshapp/src/Routing/HeaderRoutes.tsx

```diff
@@ -0,0 +1,30 @@
+import { Routes, Route } from "react-router-dom";
+import { paths } from "./RoutePaths";
+import LoggedInHeader from "../components/Header/LoggedInHeader";
+import LoggedOutHeader from "../components/Header/LoggedOutHeader";
+//Paths to render in either the logged in header or logged out header
+//logged out home has its own unique header so leaving that out for now unless we refactor
+const LOGGED_IN_PATHS = [
+  paths.logged_in_home,
+  paths.profile_page,
+  paths.profile_swipe,
+];
+const LOGGED_OUT_PATHS = [
+  //   paths.logged_out_home,
+  paths.login_page,
+  paths.forgot_password,
+  paths.sign_up,
+];
+
+export default function HeaderRoutes() {
+  return (
+    <Routes>
```

Comment:
Something seems fishy but idk how to fix, why are header and footer files essentially the same?

Reply:
Well they are pretty much doing the same thing, which is showing a specific version of the header/footer for logged in/logged out pages, but the header for the logged out home page is different from the other logged out pages, so there's a difference in routes.

Reply:
Hmmmm

Reply:
Ill come back to this